web/src/main/webapp/resources/js/atsy-candidate-create.js

```diff
@@ -244,4 +244,21 @@ function modificationDateFormatter(value, row, index) {
 function dateTimeFormatter(dateTime) {
     var options = {year: 'numeric', month: 'numeric', day: 'numeric', hour: 'numeric', minute: 'numeric', second: 'numeric'};
     return dateTime.toLocaleDateString('hu-HU', options);
-}
\ No newline at end of file
+}
+
+function actionFormatter(value, row, index) {
+    return [
+        '<a class="remove ml10 little-space" href="javascript:void(0)" title="Remove">',
```

Comment:
`title="Remove"` contains hard coded text visible to end user, thus it needs to be changed to use i18n